pkg/controller/route/route_controller_test.go

```diff
@@ -27,10 +27,10 @@ import (
 	"k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes/fake"
 	core "k8s.io/client-go/testing"
-	nodeutil "k8s.io/kubernetes/pkg/util/node"
 	"k8s.io/kubernetes/pkg/cloudprovider"
 	fakecloud "k8s.io/kubernetes/pkg/cloudprovider/providers/fake"
 	"k8s.io/kubernetes/pkg/controller"
+	nodeutil "k8s.io/kubernetes/pkg/util/node"
```

Comment:
any reasons to modify this import? Seems no need to modify this file.

Reply:
I'm a newer, Thanks. You mean import located in test file no need to modify, it can auto-modified?

Reply:
@stewart-yu I mean the original file already has imported this `nodeutil`. No need to change the import order. These imported path has already passed the `gofmt`. We are not importing any new path. So please make such unnecessary changes.

Reply:
No , i do not change the import  order .In first commit, i just modify the import name, nodeutil:/..../pkg/api/v1/node --> nodeutil:/..../pkg/util/node, but in second commit,through static code check, the import order changed, i do not know why?@dixudxan

Reply:
No , i do not change the import order .In first commit, i just modify the import name, nodeutil:/..../pkg/api/v1/node --> nodeutil:/..../pkg/util/node, but in second commit,through static code check, the import order changed, i do not know why?@dixudx

Reply:
Since the import paths are exactly the same and no new paths are added, the import order should not be changed. They have passed `gofmt` before. The order should keep unchanged.

Reply:
I have fixed it.